lua/nvim-treesitter/install.lua

```diff
@@ -411,7 +410,7 @@ end
 ---@param install_dir string
 ---@param force boolean
 ---@param generate_from_grammar boolean
-local function install_lang(lang, cache_dir, install_dir, force, generate_from_grammar)
+local function install_lang0(lang, cache_dir, install_dir, force, generate_from_grammar)
```

Comment:
Can we give this a more descriptive name (and docstring)? Something like `install_lang_(un)wrapped`?